src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -935,6 +935,40 @@ public String getUser() {
         }
     }
 
+    /**
+     * Get a Map of default tab names to deafult tab fields.
+     * The fields are returned as a String with fields separated by ;
+     * They are combined into one string in order to feed into GenFieldsCustomizerDialogViewModel.resetFields()
+     * so that they do not have to be parsed in order to fit there
+     *
+     * @param kesfjaksd;fjlkasdjflkheck.
+     * @return true sfdjklsdjfklse otherwise.
+     */
+    @Override
+    public Map<String, String> getCustomTabsNamesAndFields() {
+        Map<String, String> customTabsMap = new HashMap<>();
+
+        int defNumber = 0;
+        while (true) {
+            //Saved as CUSTOMTABNAME_def{number} and ; separated
+            String name = (String) defaults.get(CUSTOM_TAB_NAME + "_def" + defNumber);
+            String fields = (String) defaults.get(CUSTOM_TAB_FIELDS + "_def" + defNumber);
+
+            if ((name == null) || (fields == null) || name.isEmpty() || fields.isEmpty()) {
```

Comment:
You can use `StringUtil.IsNullOrEmpty` (from org.jabref.model.strings.StringUtil)